src/renderers/webxr/WebXRManager.js

```diff
@@ -238,6 +244,12 @@ class WebXRManager extends EventDispatcher {
 
 		};
 
+		this.getRenderTarget = function () {
+
+			return newRenderTarget;
+
+		};
+
```

Comment:
Should this be kept private? I don't see how this is useful in user-land with these fixes, and is otherwise an implementation detail.

Reply:
Defaulting to keeping this private.